Zend/zend_alloc.c

```diff
@@ -724,6 +727,9 @@ static void *zend_mm_chunk_alloc_int(size_t size, size_t alignment)
 		if (zend_mm_use_huge_pages) {
 			zend_mm_hugepage(ptr, size);
 		}
+#ifdef __SANITIZE_ADDRESS__
```

Comment:
nit: I think you can simplify a tad with just the [ASAN_UNPOISON_MEMORY_REGION macro](https://github.com/llvm/llvm-project/blob/6cd7500ae690b412b7a350c70b27bd8e0839d643/compiler-rt/include/sanitizer/asan_interface.h#L83)

Reply:
@devnexen What I meant is that if ASAN is unavailable, the header won't be available either, in which case we cannot include it.

Reply:
Yes I meant that in my last sentence :) we might have other usage for this macro some place else but that s fine as is tough for now :)

Reply:
I'll keep it like this for now since there isn't any other usage of `__asan_unpoison_memory_region`. I did switch to the macro though.